### DISCUSSION TITLE
Failed to open stream: No such file or directory exist

### DISCUSSION THREAD

**Discussion by mercesdes192** [chillerlan/php-qrcode#239] [Q&A]
I am new to PHP, and trying to use the php qr code and i am getting this error. Please someone help me. Thanks

( ! )  Warning:  require_once(C:\wamp64\www\qrcode\examples/../vendor/autoload.php):  Failed to open stream: No such file or directory in  C:\wamp64\www\qrcode\examples\qrcode-interactive.php on line 14

**Comment by codemasher** [comment]
Hi! Did you run `composer install` in the root directory of the repo? See also: https://php-qrcode.readthedocs.io/en/main/Usage/Installation.html#installation-with-composer
